docs/guides/mdx-on-demand.server.mdx

```diff
@@ -28,7 +28,11 @@ On the server:
 ```js path="server.js"
 import {compile} from '@mdx-js/mdx'
 
-const code = String(await compile('# hi', {outputFormat: 'function-body' /* …otherOptions */ }))
+const code = String(await compile('# hi', {
+  outputFormat: 'function-body',
+  development: false, // Avoid use of the jsxDEV runtime
```

Comment:
```suggestion
  development: false, // Generate code for production.
```

I’d suggest this: better to document what this does, instead of double negatives.

Alternatively, you can show both dev and prod versions

```suggestion
  development: false, // Generate code for production.
  // developmment: true, // Pass this to generate development code.
```

And then use the following below:

```
import * as runtime from 'react/jsx-runtime' // Production.
// import * as runtime from 'react/jsx-dev-runtime' // Development.
```